src/js/controller/controller.js

```diff
@@ -337,12 +338,26 @@ Object.assign(Controller.prototype, {
             if (model.get('playOnViewable')) {
                 if (viewable) {
                     _autoStart();
-                } else if (OS.mobile || (model.get('autoPause') && model.get('autoPause').viewability)) {
+                } else if (OS.mobile) {
                     _this.pause({ reason: 'autostart' });
                 }
             }
         }
 
+        if (_model.get('autoPause') && _model.get('autoPause').viewability) {
+            _model.set('pauseOnViewable', true);
+        }
+
+        function _checkPauseOnViewable() {
```

Comment:
`viewable` is passed in by the change callback. There is no reason to `get` it again:
`_checkPauseOnViewable(model, viewable) {`

Reply:
👍